test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -35,12 +35,13 @@
 
 public class ParserTest {
 
-    private Parser parser;
+    //private Parser parser;
```

Comment:
Generally, you should remove commented out code from your commit since VCS can help you retrieve any code from the past.